### PR TITLE
deleted var

### PR DESCRIPTION
**Pull request by Abhiram-Gullapalli**
#### Related issues

Resolves issue #16662 

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by teoli2003 — APPROVED** [review]
👍

File: files/en-us/web/api/performanceresourcetiming/redirectstart/index.md [hunk 1]
```diff
@@ -51,9 +51,9 @@ function print_start_and_end_properties(perfEntry) {
                 "responseStart", "responseEnd",
                 "secureConnectionStart"];
 
-  for (var i=0; i < properties.length; i++) {
+  for (let i=0; i < properties.length; i++) {
     // check each property
-    var supported = properties[i] in perfEntry;
+    const supported = properties[i] in perfEntry;
     if (supported) {
       var value = perfEntry[properties[i]];
```
Comment (AnilSeervi) [inline files/en-us/web/api/performanceresourcetiming/redirectstart/index.md:58]:
```suggestion
      const value = perfEntry[properties[i]];
```

File: files/en-us/web/api/performanceresourcetiming/redirectend/index.md [hunk 2]
```diff
@@ -52,9 +52,9 @@ function print_start_and_end_properties(perfEntry) {
                 "responseStart", "responseEnd",
                 "secureConnectionStart"];
 
-  for (var i=0; i < properties.length; i++) {
+  for (let i=0; i < properties.length; i++) {
     // check each property
-    var supported = properties[i] in perfEntry;
+    const supported = properties[i] in perfEntry;
     if (supported) {
       var value = perfEntry[properties[i]];
```
Comment (AnilSeervi) [inline files/en-us/web/api/performanceresourcetiming/redirectend/index.md:59]:
```suggestion
      const value = perfEntry[properties[i]];
```

File: files/en-us/web/api/performanceresourcetiming/connectend/index.md [hunk 3]
```diff
@@ -48,9 +48,9 @@ function print_start_and_end_properties(perfEntry) {
                 "responseStart", "responseEnd",
                 "secureConnectionStart"];
 
-  for (var i=0; i < properties.length; i++) {
+  for (let i=0; i < properties.length; i++) {
     // check each property
-    var supported = properties[i] in perfEntry;
+    const supported = properties[i] in perfEntry;
     if (supported) {
       var value = perfEntry[properties[i]];
```
Comment (AnilSeervi) [inline files/en-us/web/api/performanceresourcetiming/connectend/index.md:55]:
```suggestion
      const value = perfEntry[properties[i]];
```

**Review by AnilSeervi — COMMENTED** [review]
There are still places* where you can use `const`

Suggested changes*

File: files/en-us/web/api/performanceresourcetiming/connectend/index.md [hunk 4]
```diff
@@ -48,9 +48,9 @@ function print_start_and_end_properties(perfEntry) {
                 "responseStart", "responseEnd",
                 "secureConnectionStart"];
 
-  for (var i=0; i < properties.length; i++) {
+  for (let i=0; i < properties.length; i++) {
     // check each property
-    var supported = properties[i] in perfEntry;
+    const supported = properties[i] in perfEntry;
     if (supported) {
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/connectend/index.md:54]: `supported` is used only there. I find this easier to read:
```suggestion
    if (properties[i] in perfEntry) {
```

File: files/en-us/web/api/performanceresourcetiming/connectstart/index.md [hunk 5]
```diff
@@ -46,11 +46,11 @@ function print_start_and_end_properties(perfEntry) {
                 "responseStart", "responseEnd",
                 "secureConnectionStart"];
 
-  for (var i=0; i < properties.length; i++) {
+  for (let i=0; i < properties.length; i++) {
     // check each property
-    var supported = properties[i] in perfEntry;
+    const supported = properties[i] in perfEntry;
     if (supported) {
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/connectstart/index.md:52]: `supported` is used only there. I find this easier to read:
```suggestion
    if (properties[i] in perfEntry) {
```

File: files/en-us/web/api/performanceresourcetiming/fetchstart/index.md [hunk 6]
```diff
@@ -49,11 +49,11 @@ function print_start_and_end_properties(perfEntry) {
                 "responseStart", "responseEnd",
                 "secureConnectionStart"];
 
-  for (var i=0; i < properties.length; i++) {
+  for (let i=0; i < properties.length; i++) {
     // check each property
-    var supported = properties[i] in perfEntry;
+    const supported = properties[i] in perfEntry;
     if (supported) {
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/fetchstart/index.md:55]: `supported` is used only there. I find this easier to read:
```suggestion
    if (properties[i] in perfEntry) {
```

File: files/en-us/web/api/performanceresourcetiming/redirectend/index.md [hunk 7]
```diff
@@ -52,9 +52,9 @@ function print_start_and_end_properties(perfEntry) {
                 "responseStart", "responseEnd",
                 "secureConnectionStart"];
 
-  for (var i=0; i < properties.length; i++) {
+  for (let i=0; i < properties.length; i++) {
     // check each property
-    var supported = properties[i] in perfEntry;
+    const supported = properties[i] in perfEntry;
     if (supported) {
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/redirectend/index.md:58]: `supported` is used only there. I find this easier to read:
```suggestion
    if (properties[i] in perfEntry) {
```

File: files/en-us/web/api/performanceresourcetiming/redirectstart/index.md [hunk 8]
```diff
@@ -51,9 +51,9 @@ function print_start_and_end_properties(perfEntry) {
                 "responseStart", "responseEnd",
                 "secureConnectionStart"];
 
-  for (var i=0; i < properties.length; i++) {
+  for (let i=0; i < properties.length; i++) {
     // check each property
-    var supported = properties[i] in perfEntry;
+    const supported = properties[i] in perfEntry;
     if (supported) {
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/redirectstart/index.md:57]: `supported` is used only there. I find this easier to read:
```suggestion
    if (properties[i] in perfEntry) {
```

File: files/en-us/web/api/performanceresourcetiming/requeststart/index.md [hunk 9]
```diff
@@ -50,9 +50,9 @@ function print_start_and_end_properties(perfEntry) {
                 "responseStart", "responseEnd",
                 "secureConnectionStart"];
 
-  for (var i=0; i < properties.length; i++) {
+  for (let i=0; i < properties.length; i++) {
     // check each property
-    var supported = properties[i] in perfEntry;
+    const supported = properties[i] in perfEntry;
     if (supported) {
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/requeststart/index.md:56]: `supported` is used only there. I find this easier to read:
```suggestion
    if (properties[i] in perfEntry) {
```

File: files/en-us/web/api/performanceresourcetiming/responseend/index.md [hunk 10]
```diff
@@ -48,9 +48,9 @@ function print_start_and_end_properties(perfEntry) {
                 "responseStart", "responseEnd",
                 "secureConnectionStart"];
 
-  for (var i=0; i < properties.length; i++) {
+  for (let i=0; i < properties.length; i++) {
     // check each property
-    var supported = properties[i] in perfEntry;
+    const supported = properties[i] in perfEntry;
     if (supported) {
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/responseend/index.md:54]: `supported` is used only there. I find this easier to read:
```suggestion
    if (properties[i] in perfEntry) {
```

File: files/en-us/web/api/performanceresourcetiming/responsestart/index.md [hunk 11]
```diff
@@ -46,9 +46,9 @@ function print_start_and_end_properties(perfEntry) {
                 "responseStart", "responseEnd",
                 "secureConnectionStart"];
 
-  for (var i=0; i < properties.length; i++) {
+  for (let i=0; i < properties.length; i++) {
     // check each property
-    var supported = properties[i] in perfEntry;
+    const supported = properties[i] in perfEntry;
     if (supported) {
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/responsestart/index.md:52]: `supported` is used only there. I find this easier to read:
```suggestion
    if (properties[i] in perfEntry) {
```

File: files/en-us/web/api/performanceresourcetiming/secureconnectionstart/index.md [hunk 12]
```diff
@@ -49,9 +49,9 @@ function print_start_and_end_properties(perfEntry) {
                 "responseStart", "responseEnd",
                 "secureConnectionStart"];
 
-  for (var i=0; i < properties.length; i++) {
+  for (let i=0; i < properties.length; i++) {
     // check each property
-    var supported = properties[i] in perfEntry;
+    const supported = properties[i] in perfEntry;
     if (supported) {
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/secureconnectionstart/index.md:55]: `supported` is used only there. I find this easier to read:
```suggestion
    if (properties[i] in perfEntry) {
```

File: files/en-us/web/api/performanceresourcetiming/workerstart/index.md [hunk 13]
```diff
@@ -51,11 +51,11 @@ function print_start_and_end_properties(perfEntry) {
                 "secureConnectionStart",
                 "workerStart"];
 
-  for (var i=0; i < properties.length; i++) {
+  for (let i=0; i < properties.length; i++) {
     // check each property
-    var supported = properties[i] in perfEntry;
+    const supported = properties[i] in perfEntry;
     if (supported) {
```
Comment (teoli2003) [inline files/en-us/web/api/performanceresourcetiming/workerstart/index.md:57]: `supported` is used only there. I find this easier to read:
```suggestion
    if (properties[i] in perfEntry) {
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]
Thank you!

**Pull request merged**